test/gradient/elixir_fmt_test.exs

```diff
@@ -26,108 +28,151 @@ defmodule Gradient.ElixirFmtTest do
+      [expected, actual] = format_error_to_binary(errors.ret_wrong_atom2)
 
-      assert String.contains?(msg, "atom()")
-      assert String.contains?(msg, "{:ok, []}")
+      assert String.contains?(expected, "atom()")
+      assert String.contains?(actual, "{:ok, []}")
     end
 
     test "return map() instead atom()", %{wrong_ret_errors: errors} do
-      msg = format_error_to_binary(errors.ret_wrong_atom3)
+      [expected, actual] = format_error_to_binary(errors.ret_wrong_atom3)
 
-      assert String.contains?(msg, "atom()")
-      assert String.contains?(msg, "%{required(:a) => 1}")
+      assert String.contains?(expected, "atom()")
+      assert String.contains?(actual, "%{required(:a) => 1}")
     end
 
     test "return float() instead integer()", %{wrong_ret_errors: errors} do
-      msg = format_error_to_binary(errors.ret_wrong_integer)
+      [expected, actual] = format_error_to_binary(errors.ret_wrong_integer)
 
-      assert String.contains?(msg, "integer()")
-      assert String.contains?(msg, "1.0")
+      assert String.contains?(expected, "integer()")
+      assert String.contains?(actual, "float()")
     end
 
     test "return atom() instead integer()", %{wrong_ret_errors: errors} do
-      msg = format_error_to_binary(errors.ret_wrong_integer2)
+      [expected, actual] = format_error_to_binary(errors.ret_wrong_integer2)
 
-      assert String.contains?(msg, "integer()")
-      assert String.contains?(msg, ":ok")
+      assert String.contains?(expected, "integer()")
+      assert String.contains?(actual, ":ok")
     end
 
     test "return boolean() instead integer()", %{wrong_ret_errors: errors} do
-      msg = format_error_to_binary(errors.ret_wrong_integer3)
+      [expected, actual] = format_error_to_binary(errors.ret_wrong_integer3)
 
-      assert String.contains?(msg, "integer()")
-      assert String.contains?(msg, "true")
+      assert String.contains?(expected, "integer()")
+      assert String.contains?(actual, "true")
     end
 
     test "return list() instead integer()", %{wrong_ret_errors: errors} do
-      msg = format_error_to_binary(errors.ret_wrong_integer4)
+      [expected, actual] = format_error_to_binary(errors.ret_wrong_integer4)
 
-      assert String.contains?(msg, "integer()")
-      assert String.contains?(msg, "nonempty_list()")
+      assert String.contains?(expected, "integer()")
+      assert String.contains?(actual, "nonempty_list()")
     end
 
     test "return integer() out of the range()", %{wrong_ret_errors: errors} do
-      msg = format_error_to_binary(errors.ret_out_of_range_int)
+      [expected, actual] = format_error_to_binary(errors.ret_out_of_range_int)
 
-      assert String.contains?(msg, "range(1, 10)")
-      assert String.contains?(msg, "12")
+      assert String.contains?(expected, "1..10")
+      assert String.contains?(actual, "12")
+    end
+
+    test "return integer() instead float()", %{wrong_ret_errors: errors} do
```

Comment:
```suggestion
    test "return integer() instead of float()", %{wrong_ret_errors: errors} do
```

If we say that something is preferred to something else, then we use _instead of_, not just _instead_.

Reply:
Thanks :)